README.adoc

```diff
@@ -96,7 +96,7 @@ Docker communication channel
 - and manage the life cycle of the configured services
 
 For a detailed list of all available commands and their explanations refer
- to the <<commands.adoc#, Commands Document>>.
+ to the link:commands.adoc[Commands Document].
```

Comment:
`<<.., ..>> ` is [Inter-document cross references](http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#links) which probably means within the doc not out side(my guess).